static/usage.html

```diff
@@ -1188,13 +1188,13 @@ <h2><a href="#android-auto">Android Auto</a></h2>
                 permission toggles, links to related configuration screens, configuration tips, and
                 links to optional Android Auto dependencies.</p>
 
-                <p>The permission toggles ask for a confirmation before turning on. The
+                <p>The permission toggles ask for a confirmation before turning it on. The
```

Comment:
Actually, I think the original should have been "before enabling the permission."

Adding "it" makes it seem like all the toggles are doing are turning themselves on, but they're really enabling/granting permissions.

Reply:
The sentence should be written again yeah. "before enabling the permission" seems clearer to me.